jaxopt/_src/base.py

```diff
@@ -56,7 +56,7 @@ class KKTSolution(NamedTuple):
   dual_ineq: Optional[Any] = None
 
 
-N_EVAL_DTYPE = jnp.int64
+NUM_EVAL_DTYPE = jnp.int64
```

Comment:
Sorry I would prefer if we use int32. Should hopefully be my last comment :)

Reply:
The reason is because I'm worried int64 could be problematic on TPU. Plus, int32 should be enough in most use cases.

Reply:
makes sense, reverted to int64